app/models/report.rb

```diff
@@ -49,7 +49,7 @@ class Report < ActiveRecord::Base
   scope :completed, -> { where(status: %w(completed archived)) }
 
   scope :pending, lambda {
-    joins(:dispatches).where(dispatches: {status: 'pending'})
+    joins(:dispatches).where(dispatches: {status: 'pending'}).distinct
```

Comment:
what we are saying here, is that there are multiple pending dispatches for any report. Would another solution be to validate that there are no other pending dispatches before a dispatch is created?